docs/e2e/requirements.md

```diff
@@ -4,22 +4,31 @@ The following are ginkgo test focuses supported by openshift on azure e2e tests
 
 * `[EveryPR]` - These tests should be run on every PR
 * `[LongRunning]` - These tests are long running and should be run periodically
+* `[CustomerAdmin]` - These test run with `make e2e` and focus on Customer Admin capabilities
+* `[EndUser]` - These test run with `make e2e` and focus on End User capabilities
+* `[Apps]` - These test creates and validates a test application in an OpenShift cluster _(this test gets ran with `[EndUser]` focus as well)_
 
 #### Customizing e2e test scope with focuses
 
 You can specify a test focus by setting the `FOCUS` environment variable. For example
 to run all tests tagged with [EveryPR] you would specify the focus as
 
-```
+```bash
 export FOCUS=\[EveryPR\]
 ```
 
-Note: even if this document refers to focuses such as [EveryPR], you should always escape them
+or if you want to tests two focuses
+
+```bash
+ export FOCUS='\[CustomerAdmin\]|\[EndUser\]'
```

Comment:
If we use single quotes in bash, are the escapes necessary? Just trying to get a better handle around the escaping here, I'm not actually sure how the gingko focuses work :)

Reply:
I think it's because the text is reinterpreted later by more script and escaping will be necessary. Not 100% positive.